### PR TITLE
ci: add markownlint configuration file

### PR DESCRIPTION
**Pull request by romanrostislavovich**
added a `markownlint` configuration file to the project  with following rules:
- `default`: `true`
- `MD013`: `tables:false`

Closes of #72

### CODE REVIEW

File: script/gulp/task.lint.js [hunk 1]
```diff
@@ -37,7 +38,11 @@ const lintMarkdown = (done) => gulp
         config.src.exclude.ext,
     ])
     .pipe(gulpData((file) => {
-        const results = markdownlint.sync({'files': [file.path]});
+        const results = markdownlint.sync({
+            'files': [file.path],
+            'config': require(
+                path.join('../../', config.src.config.markdownlint)),
```
Comment (xferra) [inline script/gulp/task.lint.js:44]: Why is this not a part of config?

Comment (romanrostislavovich) [reply script/gulp/task.lint.js:44]: Because this file will be used  in `codeclimate`.

Comment (xferra) [reply script/gulp/task.lint.js:44]: I mean why do we need `../../`?

**Review by xferra — APPROVED** [review]
Approved for now, let's refactor config and path-related mess later

**Pull request merged**